data/tests/tests_wormhole_navigation.txt

```diff
@@ -350,87 +350,179 @@ test "Tests - Ember Wastes Navigation"
 			travel Terminus
 			travel Limen
 			travel Mora
+		# Our test-savegames don't have the date conditions serialized. Need to
+		# have passed at least one date-transition to get the date conditions set
+		# (until PR https://github.com/endless-sky/endless-sky/pull/5244 removes
+		# this need to serialize the date conditions completely).
+		#assert
+		#	year == 3012
+		#	month == 12
+		#	day == 16
```

Comment:
As a future improvement to the integration testing framework, consider how you might identify _which assert_ in a group of them did not pass.

Getting the specific assert text tokens will probably be easier in the generic condition case than getting the substituted token values, and is likely "enough" when combined with the associated failure context to deduce the problem.

Reply:
Yes, your other idea of printing the values of the variables used in the assert/condition could also be helpful here (since currently only the "test: ..." conditions get printed). Still on the todo list somewhere.